frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/common.embedding.unit.spec.tsx

```diff
@@ -100,37 +149,142 @@ describe("[OSS] embedding settings", () => {
-      expect(location.pathname).toEqual(staticEmbeddingSettingsUrl);
+        expect(
+          withinStaticEmbeddingCard.getByRole("heading", {
+            name: "Static embedding",
+          }),
+        ).toBeInTheDocument();
+        expect(
+          withinStaticEmbeddingCard.getByText(/Use static embedding when/),
+        ).toBeInTheDocument();
+      });
+
+      it("should not allow access to the static embedding settings page", async () => {
+        // Go to static embedding settings page
+        await userEvent.click(
+          within(
+            screen.getByRole("article", {
+              name: "Static embedding",
+            }),
+          ).getByRole("button", { name: "Manage" }),
+        );
+
+        expect(screen.getByText("Embedding secret key")).toBeInTheDocument();
+      });
     });
 
-    it("should not allow going to interactive embedding settings page", async () => {
-      const { history } = await setupEmbedding({
-        settingValues: { "enable-embedding": true },
+    describe("embedding SDK", () => {
+      it("should show info about embedding SDK", async () => {
+        const withinEmbeddingSdkCard = within(
+          screen.getByRole("article", {
+            name: "Embedding SDK for React",
+          }),
+        );
+
+        expect(
+          withinEmbeddingSdkCard.getByRole("heading", {
+            name: "Embedding SDK for React",
+          }),
+        ).toBeInTheDocument();
+        expect(
+          withinEmbeddingSdkCard.getByText(
+            /Interactive embedding with full, granular control./,
+          ),
+        ).toBeInTheDocument();
       });
 
-      expect(
-        screen.queryByRole("button", { name: "Configure" }),
-      ).not.toBeInTheDocument();
-      expect(
-        screen.getByRole("link", { name: "Learn More" }),
-      ).toBeInTheDocument();
+      it("should allow access to the embedding SDK settings page", async () => {
+        // Go to embedding SDK settings page
+        await userEvent.click(
+          within(
+            screen.getByRole("article", {
+              name: "Embedding SDK for React",
+            }),
+          ).getByRole("button", { name: "Try it out" }),
+        );
+
+        expect(
+          screen.getByLabelText("Cross-Origin Resource Sharing (CORS)"),
+        ).toBeDisabled();
+      });
+    });
+
+    describe("interactive embedding", () => {
+      it("should not allow going to interactive embedding settings page", async () => {
+        expect(
+          screen.queryByRole("button", { name: "Configure" }),
+        ).not.toBeInTheDocument();
+        expect(
+          screen.getByRole("link", { name: "Learn More" }),
+        ).toBeInTheDocument();
+
+        act(() => {
+          history.push(interactiveEmbeddingSettingsUrl);
+        });
 
-      act(() => {
-        history.push(interactiveEmbeddingSettingsUrl);
+        expect(history.getCurrentLocation().pathname).toEqual(
+          embeddingSettingsUrl,
+        );
```

Comment:
I wonder why we're being redirected back to the embeddingSettingsUrl, even if we cannot visit interactiveEmbeddingSettingsUrl. Is this because of missing license, or because interactive embedding is disabled?

Reply:
It's true that we can't visit via the UI, but users can still enter the page directly via the URL.

So we have this test, we test that if we visited the URL directly e.g. `/admin/settings/embedding-in-other-applications/full-app` users won't be able to see the setting page.

This is the old test by the way.